hbase-server/src/main/java/org/apache/hadoop/hbase/procedure2/store/region/RegionProcedureStore.java

```diff
@@ -382,7 +383,8 @@ public void recoverLease() throws IOException {
     if (conf.get(USE_HSYNC_KEY) != null) {
       conf.set(HRegion.WAL_HSYNC_CONF_KEY, conf.get(USE_HSYNC_KEY));
     }
-    conf.setInt(AbstractFSWAL.RING_BUFFER_SLOT_COUNT, IntMath.ceilingPowerOfTwo(16 * numThreads));
+    conf.setInt(AbstractFSWAL.RING_BUFFER_SLOT_COUNT,
```

Comment:
emmmm, why this format change? What is the max line width in your formatter config?